modules/root/root_api.rb

```diff
@@ -19,7 +18,9 @@ class Proxy::RootApi < Sinatra::Base
 
   get "/version" do
     begin
-      {:version => Proxy::VERSION}.to_json
+      content_type :json
+      modules = Hash[::Proxy::Plugins.enabled_plugins.collect {|plugin| [plugin.plugin_name.to_s, plugin.version.to_s]}].reject { |key| key == 'foreman_proxy' }
+      {:version => Proxy::VERSION, :modules => modules}.to_json
```

Comment:
Adding another key to this hash shouldn't break existing clients: they expect hash, but are only aware of :version key, which remains. Newer clients can use both keys.

We should probably add versioning to api; if anything, this would give plugin authors more freedom/flexibility to make breaking changes in their api.